### DISCUSSION TITLE
How to resize thumbnail previews in startpage?

### DISCUSSION THREAD

**Discussion by 7eopard** [sumatrapdfreader/sumatrapdf#3614] [Documentation]
I've read advance setting's docs, but failed to did it.

When a pdf or epub file is opened and shown in startpage, it would be way better if theirs' preview is "complete", that is, in protrait view. Thinking I'm looking for a way to do so.

**Comment by GitHubRulesOK** [comment]
It is a fixed ratio ? 212 x 150? so only way to fool it is change the files inbuilt cover to landscape/upper half of portrait, or edit the SumatraPDF sources

**Reply by 7eopard** [reply]
Now some preview are stretched, some titles are hidden. Sad.

**Reply by GitHubRulesOK** [reply]
they should not be "stretched" it is   212  x 150 depending on source scale
Agreed they can be less useful when its a folder icon or the title is at bottom of page
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/38e1a67f-afcb-413c-b34e-414522c5fbbe)

So it is possible to put your own image in place of the stored one **HOWEVER** it may be replaced by another copy of the old one later so you need to force file / folder to read only (not a good idea)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/78752b78-121a-434a-8970-050a2d9133a0)
